monkey/monkey_island/cc/ui/src/components/pages/RunMonkeyPage/commands/local_windows_powershell.js

```diff
@@ -1,15 +1,20 @@
 import {OS_TYPES} from '../utils/OsTypes';
 
 
-export default function generateLocalWindowsPowershell(ip, osType, username) {
+export function getAgentDownloadCommand(ip, osType) {
```

Comment:
If we're renaming this function here, let's do it in the other two places too.

Reply:
This function is still here, I simply extracted the command formatting into a separate function

Reply:
Oh. 🤦‍♀️ My bad.